docs: useUserOrder docs update (#360)
vuestorefront/bigcommerce: docs/composables/use-user-order.md

## Features

> `useUserOrder` composable is used to fetch customer orders.

Reviewer: ```suggestion
The `useUserOrder` composable is used for fetching customer orders.
```

## API

> - `search` - function for fetching orders from the API and populating the `orders` object with the result. This method accepts a single object parameter. See the [GetOrdersParameters](../reference/api/bigcommerce-api.getordersparameters.md) interface for more information.

Reviewer: ```suggestion
- `search` - a function for fetching orders from the API and populating the `orders` object with the returned data. This method accepts a single object parameter. See the [GetOrdersParameters](../reference/api/bigcommerce-api.getordersparameters.md) interface for more information.
```

> - `orders: OrderResults` - the main data object. See [OrderResults](../reference/api/bigcommerce.orderresults.md) interface for more information.

Reviewer: ```suggestion
- `orders: OrderResults` - the main data object. See the [OrderResults](../reference/api/bigcommerce.orderresults.md) interface for more information.
```

> - `loading: boolean` - a reactive object containing information about the loading state of `search` method.

Reviewer: ```suggestion
- `loading: boolean` - a reactive object containing information about the loading state of the `search` method.
```

> - `error: UseUserOrderErrors` - reactive object containing the error message, if `search` failed for any reason. See the [UseUserOrderErrors](https://docs.vuestorefront.io/v2/reference/api/core.useuserordererrors.html#useuserordererrors-interface) interface for more information.

Reviewer: ```suggestion
- `error: UseUserOrderErrors` - a reactive object containing the error message, if the `search` failed for any reason. See the [UseUserOrderErrors](https://docs.vuestorefront.io/v2/reference/api/core.useuserordererrors.html#useuserordererrors-interface) interface for more information.
```
